draft-ietf-ppm-dap.md

```diff
@@ -1670,6 +1676,7 @@ enum {
   task_expired(7),
   invalid_message(8),
   report_too_early(9),
+  task_not_started(10),
```

Comment:
My initial though was that we may be able to use `report_too_early` for this case, but looking at how that error currently used, I think I'm in favor of adding `task_not_started`. The reason is that rejecting a report rejected with `report_too_early` can be retried, but if the report's timestamp precedes the task start time, then there is no way to process it.

WDYT @branlwyd?

Reply:
I agree; as you note, there are semantics around `report_too_early` that are not appropriate to apply to the cases we will be using `task_not_started`.